packages/bitcore-wallet-service/src/lib/model/email.ts

```diff
@@ -1,24 +1,6 @@
 import _ from 'lodash';
+import Uuid from 'uuid';
 
-const Uuid = require('uuid');
-
-interface IEmail {
```

Comment:
In other models (e.g. wallet) we've used this interface to help define the class below. e.g.
```
export class Email implements IEmail {
```

There may be some discrepancies between the Email class and the IEmail interface. In the case of the Wallet class, I had to do some extra workaround stuff, so if this email model file gets to be too much of a change then you can just punt on it for now and eslint ignore the no unused var rule for the IEmail interface.